apps/src/common-styles.module.scss

```diff
@@ -21,6 +21,29 @@
     background-color: color.$lighter_purple;
     color: color.$dark_charcoal;
   }
+
+  button {
```

Comment:
Agree, It might be a good idea to add a classname for paneButton specifically.
However, pls note that this styles will only affect buttons from .purpleheader

Reply:
If it's useful, you could use the `remove-button-styles` mixin from apps/src/mixins.scss to remove the default button styling and apply whatever remaining custom styles you need.

Reply:
Moved this styles to apps/src/templates/pane-header.module.scss, turned out there's a className for paneButton already